README.md

```diff
@@ -129,6 +129,19 @@ SDK also includes a Metadata API to interact with AppData documents and IPFS CID
   // Decode AppData Hex to CID
   const decodedAppDataHex  = await cowSdk.metadataApi.appDataHexToCid(hash)
   console.log(decodedAppDataHex) //QmUf2TrpSANVXdgcYfAAACe6kg551cY3rAemB7xfEMjYvs
+
+  // Create an AppData Document
+    const appDataDoc = cowSdk.metadataApi.generateAppDataDoc({})
+    /* {
+        version: '0.1.0',
+        appCode: 'CowSwap',
+        metadata: {},
+      } */
+
+  // Upload AppDataDoc to IPFS
+   const cowSdk = new CowSdk(4, { ipfs: { apiKey: 'YOUR_API_KEY', apiSecret: 'YOUR_API_SECRET' } })
```

Comment:
As @alfetopito mentioned below, these keys are exclusively from Pinata but we are not mentioning it anywhere